test/cmake/seqan3_require_benchmark.cmake

```diff
@@ -48,3 +46,29 @@ macro (seqan3_require_benchmark)
 
     unset (gbenchmark_path)
 endmacro ()
+
+macro (seqan3_require_benchmark)
+    enable_testing ()
+
+    set (gbenchmark_git_tag "v1.5.0")
+
+    if (NOT ${CMAKE_VERSION} VERSION_LESS 3.14)
```

Comment:
cmake 3.7 is still support, but cmake 3.14 or higher will use the new method. This makes it easier as a transition for our members.

Reply:
Jup, I skipped the line when reading the changes...

Still, macOS cmake has some problem with it...

Reply:
The problem was that the ci script does the bechmark/googletest install step during configuration, but the target that we used to express that don't always exist any more. This also means that we use on mac the default cmake installation and not the one we download.